### PR TITLE
Map integration

### PR DESCRIPTION
**Pull request by nico-rodriguez**

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import Footer from './home/components/Footer';
 import Header from './home/components/Header';
-import global from './home/css/global.css';
+import MapContainer from './home/components/MapContainer';
+import Wrapper from './home/components/Wrapper';
+import BodyWrapper from './home/components/BodyWrapper';
+import global from './home/css/global.css'; //eslint-disable-line
```
Comment (fbalarini) [inline src/app/App.js]: Please remove `eslint-disable-line`

File: src/app/home/components/MapContainer.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactMapboxGl from 'react-mapbox-gl';
+import mapboxgl from 'mapbox-gl';
+import MapComponent from './MapComponent';
+
+const Map = ReactMapboxGl({
+  accessToken: 'pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4M29iazA2Z2gycXA4N2pmbDZmangifQ.-g_vE53SD2WrJ6tFX7QHmA',
```
Comment (fbalarini) [inline src/app/home/components/MapContainer.js]: I think this should be an env variable, what do you think?

File: src/app/App.js (hunk 1 above)
Comment (nico-rodriguez) [reply src/app/App.js]: Done.

File: src/app/home/components/MapContainer.js (hunk 2 above)
Comment (nico-rodriguez) [reply src/app/home/components/MapContainer.js]: I agree. Moved the token to an env variable inside .env file.

**Review by fbalarini — APPROVED** [review]

File: src/app/home/components/Wrapper.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
```
Comment (alexisartus96) [inline src/app/home/components/Wrapper.js]: Is this file necessary?

**Review by alexisartus96 — COMMENTED** [review]
Please move style files to "style" folder.

File: src/app/home/components/Wrapper.js (hunk 3 above)
Comment (nico-rodriguez) [reply src/app/home/components/Wrapper.js]: It's not necessary. Already removed it.
